src/mixin.scss

```diff
@@ -0,0 +1,84 @@
+@mixin questionMark($position, $top, $right, $width, $height) {
```

Comment:
CSS of the question mark is the same for all components so, there is no need to pass the arguments.
Sama applies for the error messages, descriptions and others.